db/migrate/20211117094502_add_missing_taggable_index.acts_as_taggable_on_engine.rb

```diff
@@ -0,0 +1,13 @@
+# This migration comes from acts_as_taggable_on_engine (originally 4)
+class AddMissingTaggableIndex < ActiveRecord::Migration[5.0]; end
+AddMissingTaggableIndex.class_eval do
+  # disable_ddl_transaction! if System::Database.postgres?
+
+  def self.up
+    rename_index ActsAsTaggableOn.taggings_table, 'index_taggings_on_taggable_id_and_taggable_type_and_context', 'taggings_taggable_context_idx'
```

Comment:
So this rename is to be inline with the upstream naming right?

Reply:
yes